schema/field.js

```diff
@@ -1,2 +1,2 @@
-import { valueSchemas } from '@mapeo/schema'
-export const fieldSchema = valueSchemas['field']
+import { valueSchemas } from '@comapeo/schema'
+export const fieldSchema = valueSchemas.field
```

Comment:
This line was changed by the `standard` auto-formatter we have in this repo. (We should probably update this to Prettier at some point.)